pyfarm/agent/http/api/assign.py

```diff
@@ -187,15 +187,15 @@ def assignment_started(_, assign_id):
 
         def assignment_stopped(_, assign_id):
             logger.debug("Assignment %s has stopped", assign_id)
+            if (len(config["current_assignments"]) <= 1 and
```

Comment:
No, because we do this immediately before removing the current assignment from config["current_assignments"], so it will always be at least 1.